sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/aggregate/ApproximatePercentileSuite.scala

```diff
@@ -169,40 +167,40 @@ class ApproximatePercentileSuite extends SparkFunSuite {
     val defaultAccuracy = ApproximatePercentile.DEFAULT_PERCENTILE_ACCURACY
     // sql, single percentile
     assertEqual(
-      s"percentile_approx(`a`, 0.5D, $defaultAccuracy)",
+      s"percentile_approx(`a`, 0.5D, ${defaultAccuracy}L)",
```

Comment:
do we still need this change?

Reply:
yes, we still need this, this `defaultAccuracy` is a long